src/View/Helper/FlashMessenger.php

```diff
@@ -267,9 +285,14 @@ public function getMessageCloseString()
      * @param  string $messageOpenFormat
      * @return FlashMessenger
      */
-    public function setMessageOpenFormat($messageOpenFormat)
+    public function setMessageOpenFormat($messageOpenFormat, string $namespace = 'default')
     {
-        $this->messageOpenFormat = (string) $messageOpenFormat;
+        $objNamespace = $this->getNamespace($namespace);
+        if ($objNamespace !== null) {
```

Comment:
Similar to other locations in the patch:

 * `else` can be omitted when redundant
 * write the positive conditional first, rather than adding a negation (reduces mental overhead when reading)